node/src/main/resources/defaults.conf

```diff
@@ -0,0 +1,366 @@
+  # See map_size argument definition in https://lmdb.readthedocs.io/en/release/#environment-class
+
+  # RSpace storage
+  lmdb-map-size-rspace = 10T
+  # Blockdagstore indexes
+  lmdb-map-size-blockdagstore = 8G
+  # Blockstore indexes
+  lmdb-map-size-blockstore = 8G
+  # Deploys storage
+  lmdb-map-size-deploystore = 1G
+}
+
+tls {
+  # "Path to X.509 certificate for TLS. If file does not exist, certificate will be
+  # generated from PEM key located at `tls.key-path`.
+  certificate-path = ${storage.data-dir}"/node.certificate.pem"
+
+  # Path to private key for TLS. Elliptic curve secp256r1 key in PEM format is supported.
+  # If file does not exist, new key will be generated.
+  key-path = ${storage.data-dir}/node.key.pem
+
+  # Use a non blocking secure random instance to generate TLS key
+  secure-random-non-blocking = false
+
+  # TODO possibly remove
+  custom-certificate-location = false
+  custom-key-location = false
+}
+
+casper {
+  # Block is considered as finalized if its fault tolerance is bigger then this value. For more info
+  # https://github.com/rchain/rchain/blob/dev/casper/src/main/scala/coop/rchain/casper/SafetyOracle.scala
+  fault-tolerance-threshold = 0.0
+
+  # Base16 encoding of the public key to use for signing a proposed blocks.
+  # Can be inferred from the private key for some signature algorithms.
+  # validator-public-key =
+
+  # Base16 encoding of the private key to use for signing a proposed blocks.
+  # It is not recommended to use in production since private key could be revealed through the process table.
+  # validator-private-key =
+
+  # Path to the base16 encoded private key to use for signing a proposed blocks.
+  # validator-private-key-path =
+
+  # Shard name written into PoS contract during genesis.
+  shard-name = "root",
+
+  # ID of the parent shard. For root shard it's `/`. ID if the shard is `{parent-shard-id}/{shard-name}`.
+  # So for root named `root` shard ID is `/root`, for its child shard `rootchild` shard ID is `/root/rootchild`.
+  parent-shard-id = "/"
+
+  # Interval of the casper loop to maintain requested blocks and missing dependent blocks
+  # in the node runtime, there is a loop which would fetch dependency of the missing dag and maintain
+  # requested blocks. See
+  # https://github.com/rchain/rchain/blob/43a1dfd90ec16a4a5c3b24e7d7aab8bc160ccb2b/node/src/main/scala/coop/rchain/node/NodeRuntime.scala#L746-L759
+  casper-loop-interval = 30 seconds
+
+  # Timeout for the requested blocks
+  # all requests sent to request blocks is put into a data structure for maintain.
+  requested-blocks-timeout = 240 seconds
+
+  # Finalization is called every `finalization-rate` blocks
+  finalization-rate = 1
+
+  # Maximum number of block parents
+  max-number-of-parents = 2147483647
+
+  # Maximum depth of block parents. https://github.com/rchain/rchain/pull/2816
+  # Limits the depth of secondary parents in regards to the depth of the main parent.
+  max-parent-depth = 2147483647
```

Comment:
Just curious. How do you get this number? I saw 7-depth in the link you posted

Reply:
I just used `max-number-of-parents` value as `some random big number` to make effectively `max-parent-depth` effectively unlimited. This option is experimental and I we don't have any justification for why it's needed and what number this should be. So I made it unlimited by default.